metrics/metrics.go

```diff
@@ -0,0 +1,28 @@
+package metrics
```

Comment:
Can you pls add https://github.com/red-hat-storage/odf-operator/blob/main/hack/boilerplate.go.txt